pkg/collector/corechecks/cluster/helm/helm.go

```diff
@@ -227,6 +231,15 @@ func tagsForMetricsAndEvents(release *release, storageDriver helmStorage, includ
 		tags = append(tags, fmt.Sprintf("helm_status:%s", release.Info.Status))
 	}
 
+	for configParam, tagName := range configParamsAsTags {
+		value, err := release.getConfigValue(configParam)
```

Comment:
I think this function call can be resource-intensive, maybe it's worth caching the tags and only recall it when a new version is detected?

Reply:
That's a good point. I think that's something that we can apply for all the tags and not just the ones added in this PR.
I've taken a look and I think all the tags associated with a release are immutable except for `helm_status`.

Reply:
I added a new commit with this :+1: